docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1325,6 +1325,12 @@ modes.
 * Legacy
 * Experimental
 
+You should migrate any existing Hive views that you want to access from Trino to
```

Comment:
Sure .. but from the perspective of a Trino user .. if want things to work well. you should use Trino views, Hive views only sort of work (via translation) .. we should say that

Reply:
So how do we want to reword.. 

If you want to access Hive views from Trino, it is best to migrate them to Trino views.

or

It is best to migrate Hive views to Trino views to avoid translation and potential differences in results between accessing the Hive views from Trino and other engines.

Reply:
The only information I currently see in https://trino.io/docs/current/connector/hive.html  which is remotely reffering to Trino views is:

> [View management](https://trino.io/docs/current/language/sql-support.html#sql-view-management); see also [Hive-specific view management](https://trino.io/docs/current/connector/hive.html#hive-sql-view-management)

There is no explicit content though in the docs of the Hive connector though about _Trino views_ .

How about having a section dedicated for Trino views with explicit examples on how to create / drop a Trino view ?
Maybe, going a step forward and pointing out tha the Trino views can federate data across multiple connectors.

Reply:
Yes ... that would be a good additional thing to do .. but would have to cover multiple connectors and potentially the SQL docs .. so I suggest we do that in a separate PR .. specifically after converting these docs to markdown and also getting the refactor from @m57lyra about file formats and metastores in..